### ISSUE TITLE
Add support for tagged templates

### ISSUE THREAD

**Issue by amk221** [netguru/eslint-plugin-ember#62]
```javascript
Component.extend({
  layout: hbs`Hello World`,
  init() {},
  actions: {}
});
```

`layout` is being detected as an unknown type, and so I am getting suggestions it should be placed lower down.

**Comment by michalsnik** [comment]
I think you're right and we should support it too 👍
